PORTAL/jobs/requests.py

```diff
@@ -40,24 +40,26 @@ def from_raw(cls, raw: Any):
     @classmethod
     def parse(cls, idstr: str):
         kinds = '|'.join(cls._KIND_BY_VALUE)
-        m = re.match(rf'^req-(?:({kinds})-)?(\d{{10}})-(\w+)$', idstr)
+        m = re.match(rf'^req-(?:({kinds})-)?(\d{{10}})-(\w+)-(\w+)$', idstr)
```

Comment:
What about all the existing requests that don't have a worker ID in the request ID?

Reply:
Done